packages/now-node/test/fixtures/48-change-handler-by-env/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (req, res) => {
+  res.end('default handler');
+}
+
+exports.myCustomHandler = async function(req, res) {
+  res.end('custom handler');
```

Comment:
There is no `res` object here for the AWS API.

See test fixture `25-aws-api` for example usage.

```
exports.handler = async function () {
  return {
    statusCode: 200,
    headers: {},
    body: 'custom handler',
  };
};
```

Reply:
Thanks, implemented it